src/accessor/tick.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+module.exports = function() {
+  var date = function(d) { return d.date; },
+      high = function(d) { return d.high; },
+      low = function(d) { return d.low; },
+      spread = function(d) { return d.spread; };
```

Comment:
This doesn't seem to be used in the plot (and also can be derived), should it be removed?